### PR TITLE
Fix: Frontend fix for sorting unpaid bounties first

### LINKED ISSUE #1274: Add parameters to prioritize open bounties inside of profile for assigned bounties
### Context
Currently, bounties are listed chronologically regardless of status.
Bounties

We've gotten feedback from several hunters who said they would prefer to have open bounties first as it would help the locate bounties they need to work on first.

As a bounty provider, it would also help me keep track of open bounties as well. 

The fix is to add a parameter to the api call to prioritize open bounties call first. I'll create another issue to enable pagination in the backend to match the 20 displayed bounties under assigned bounties.

### Design
One simple and quick fix would be to simply show open bounties first like this:
Screenshot 2024-01-06 at 10 32 25 AM

### Acceptance Criteria
- [ ] I've tested on Chrome

### PR DESCRIPTION
**Pull request by VividhPandey003**
## Describe your changes
Adds a fix from the frontend to display the unpaid bounties first under the user-> assigned bounties
## Type of change
Might close #1274
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

File: frontend/app/src/store/main.ts [hunk 1]
```diff
@@ -913,6 +913,16 @@ export class MainStore {
             organization: { ...organization }
           });
         }
+        // Sort based on the "paid" property
+        ps3.sort((a: any, b: any) => {
```
Comment (elraphty) [inline frontend/app/src/store/main.ts]: Please remove the array sort function, the sort is being done from the database now, thanks

**Review by elraphty — CHANGES_REQUESTED** [review]

**Comment by elraphty** [comment]
Please fix merge conflicts @VividhPandey003

**Comment by VividhPandey003** [comment]
> Please fix merge conflicts @VividhPandey003

on it

File: frontend/app/src/store/main.ts (hunk 1 above)
Comment (VividhPandey003) [reply frontend/app/src/store/main.ts]: Already removed in the second commit

File: frontend/app/src/store/main.ts [hunk 2]
```diff
@@ -885,8 +885,8 @@ export class MainStore {
     queryParams = { ...queryParams, search: uiStore.searchText };
 
     const query = this.appendQueryParams(`people/wanteds/assigned/${pubkey}`, queryLimit, {
-      ...queryParams,
-      sortBy: 'created'
+      sortBy: 'created',
```
[the remark this answers is not in the document]
Comment (VividhPandey003) [reply frontend/app/src/store/main.ts:888]: okay

**Comment by ecurrencyhodler** [comment]
Paid! https://community.sphinx.chat/bounty/1163

**Pull request merged**